src/lib.rs

```diff
@@ -59,6 +60,33 @@ pub enum DisplayState {
     Editable,
 }
 
+/// Configure spacing and separators for the view
```

Comment:
I would like to see this commented extended. Please take a look at https://github.com/hellow554/cursive_hexview/blob/4168369ea2ff121520fa1a2b789ba2c0bdef4d40/src/lib.rs#L45-L60 for what I mean

Reply:
Can you give me any direction or suggestion? I thought with the one-liners for each of the individual fields and the descriptive field names would be clear.

Reply:
```
Controls the visual output of the `HexView` struct.

There are various options which can be altered. For a detailed description of them, please see the fields below.
For the changes to apply, you need to use [`set_config`].

[`set_config`]: struct.HexView.html#method.set_config
```